enclave_build/src/docker.rs

```diff
@@ -14,6 +14,10 @@ use tempfile::NamedTempFile;
 use tokio::prelude::{Future, Stream};
 use url::Url;
 
+/// Docker inspect architecture constants
+pub const DOCKER_ARCH_ARM64: &str = "arm64";
+pub const DOCKER_ARCH_AMD64: &str = "amd64";
```

Comment:
I'd go for the more commonly used aarch64 and x86_64 instead, especially since it's difficult to see the difference between arm64 and amd64.

Reply:
That was the intention but `docker inspect` returns these constants.